lib/modules/manager/gradle/parser.spec.ts

```diff
@@ -418,6 +419,24 @@ describe('modules/manager/gradle/parser', () => {
       });
     });
 
+    describe('properties map accessors', () => {
```

Comment:
The test for `properties` is redundant with the test above, `project` and `rootProject` prefixes are also tested in `common.spec.ts`. So I'd suggest you leave out this test block entirely, as it doesn't test behavior that wouldn't be covered elsewhere.

Reply:
Makes sense @Churro , I'll leave this out

Reply:
Removed, but lost the approval 😬